server/src/db/index.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+function connect() {
+  return new Promise((resolve, reject) => {
+    if (process.env.NODE_ENV === "test") {
+      mongoose
+        .connect(
+          global.__MONGO_URI__,
+          { useNewUrlParser: true, useCreateIndex: true },
+          err => {
+            if (err) {
+              console.error(err);
+              process.exit(1);
+            }
+          }
+        )
+        .then((res, err) => {
+          if (err) return reject(err);
+          resolve();
+        });
```

Comment:
Instead of creating a new promise, and resolving/rejecting it depending on another promise, do you reckon we can simplify this `connect` function by directly returning the promise from the `connect()`?

Or will that result in different behaviour?

Reply:
Done. Did a few tests and it looks like the behavior is the same :)